pkgs/applications/graphics/pinta/default.nix

```diff
@@ -1,14 +1,18 @@
-{ lib, fetchFromGitHub, buildDotnetPackage, dotnetPackages, gtksharp,
-  gettext }:
+{ lib
+, fetchFromGitHub
+, buildDotnetPackage
+, dotnetPackages
+, gtksharp
+, gettext
+}:
 
 let
   mono-addins = dotnetPackages.MonoAddins;
 in
 buildDotnetPackage rec {
-  name = "pinta-1.6";
-
   baseName = "Pinta";
```

Comment:
Is this exposed to the package? Shouldn't this be a `let ... in` instead?

Reply:
see: https://github.com/NixOS/nixpkgs/pull/144949#discussion_r744269519 . `baseName` will eventually be renamed to `pname`